pom.xml

```diff
@@ -67,7 +67,7 @@
         <tag>HEAD</tag>
     </scm>
     <properties>
-        <jackson.version>2.6.5</jackson.version>
+        <jackson.version>2.8.9</jackson.version>
```

Comment:
This version update is a good idea, but according to our revapi check it introduces some incompatible changes. I'm fine with them, but I will explicitly mention this version update in our release notes.

Reply:
Yep i was also concerned about all the revapi complains, this is actually worth mentioning this in the release since it might create problems for people using jackson transitively from our API.